spec/preconfirmation-notes.md

```
@@ -0,0 +1,47 @@
+
+- integrate with look-ahead window from relay / beacon node (?)
+- chainid - hoodi
+- only support commitment type 1
+
+`fee`
+
+- r eth node
+- [For our pilot and sample pricing rule, what we can do is increase the marginal price of a new preconf as a function of the blockspace share already used.
+E.g. this curve ensures there's always space left:
+`tx_price = reth_oracle_price / (1 - (preconfed_gas / gas_limit)^k`](https://www.notion.so/For-our-pilot-and-sample-pricing-rule-what-we-can-do-is-increase-the-marginal-price-of-a-new-precon-264ced2a7e1380b58ff2c8a4d42c62fe?pvs=21)
+    - k - some unknown value
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)